templates/drupal-php/init/50-deploy-changes.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o pipefail
+set -o nounset
+
+if [[ -n "${DEBUG}" ]]; then
+    set -o xtrace
+fi
+
+# Exit cleanly if the "DRUPAL_DEPLOY_CHANGES" env var is not set to "auto".
+if [[ "${DRUPAL_DEPLOY_CHANGES:}" != "auto" ]]; then
+  echo "Skipping automatic drupal deploy."
+  exit 0
+else
+  echo "Starting automatic drupal deploy."
+fi
+
+drsh() {
+  drush --root="${DRUPAL_ROOT}" --uri="${PROJECT_BASE_URL:-}" ${@}
+}
+
+run_query() {
+  $(drsh sql:connect) -s -N -e "${1}"
+}
+
+get_db_version() {
+  local drupal_db_version_key_name="db-schema-version"
+
+  echo $(run_query "SELECT value FROM key_value WHERE collection='frontkom' AND name='${drupal_db_version_key_name}'")
+}
+
+main() {
+  deployment_identifier_file="${APP_ROOT}/.VERSION"
```

Comment:
This is the part which would need to be looked at because the `.VERSION` is generated by the [generate-deployment-identifier command](https://github.com/nymedia/generate-deployment-identifier), which is only taking care of the composer changes, nothing to do with Drupal.

A better way would be probably to compute a hash of site-schema.json to ensure we deploy each time there are new db updates or new modules as well. 

At the same time using only site-schema does not take in account changes to drupal config only.. 
So ultimately you should look into finding a new method to detect if changes should be deployed or not.